src/Arcus.Templates.ServiceBus.Topic/Arcus.Templates.ServiceBus.Topic.csproj

```diff
@@ -56,7 +56,6 @@
     <PackageReference Include="Serilog.Extensions.Hosting" Version="3.0.0" Condition="'$(ExcludeSerilog)' == 'false'" />
     <PackageReference Include="Serilog.Sinks.ApplicationInsights" Version="3.1.0" Condition="'$(ExcludeSerilog)' == 'false'" />
     <PackageReference Include="Serilog.Sinks.Console" Version="3.1.1" Condition="'$(Configuration)' == 'Debug'" />
-    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.9.10" />
```

Comment:
It's not a removal, apperently, the Container tools package was duplicated here.